src/dct/ai/Mission.lua

```diff
@@ -153,11 +153,11 @@ function PrepState:timeextend(addtime)
 	self.timer:extend(addtime)
 end
 
-local function composeBriefing(msn, tgt)
+local function composeBriefing(msn, tgt, start_time)
 	local briefing = tgt.briefing
 	local interptbl = {
 		["TOT"] = os.date("%F %Rz",
-			dctutils.zulutime(msn:getTimeout()*.6)),
+			dctutils.zulutime(start_time + MISSION_LIMIT * 0.6)),
```

Comment:
This might be a problem with the mission states now, as the time on target is variable based on when the mission transitions to the active state. Also I am not sure if anyone uses TOT really we could just delete it.

Reply:
TOT is verified on an old test, so this change was necessary after fixing the time formatter. For compatibility, this could be kept as a best effort (it assumes 60% of the mission time with zero prep) until the TOT feature is implemented properly, as outright removal might require changing the test case to not use it in the briefing.